public/styles.css

```diff
@@ -208,3 +203,38 @@ input {
   align-items: center;
   height: 100vh;
 } */
+
+.homepage {
+  display: none;
```

Comment:
probably should not be hidden since it's the main page. The modal should be hidden instead...